build.gradle

```diff
@@ -127,6 +128,38 @@ jacocoTestReport {
 
 project.tasks['sonarqube'].dependsOn jacocoTestReport
 
+pitest {
+  group = "Verification"
+  pitestVersion.set('1.5.1')
+  junit5PluginVersion.set('0.12')
+  testSourceSets = [sourceSets.test]
+  mainSourceSets = [sourceSets.main]
+  targetClasses = ['uk.gov.hmcts.reform.bulkscan.orchestrator.*']
+  excludedClasses = [
+    'uk.gov.hmcts.reform.bulkscan.orchestrator.config.*'
+  ]
+  threads = 10
+  enableDefaultIncrementalAnalysis = true
+  outputFormats = ['XML', 'HTML']
+  timestampedReports = false
+  mutationThreshold = 50
+  timeoutConstInMillis = 30000
+  mutators.set([
+    'CONDITIONALS_BOUNDARY',
+    'INCREMENTS',
+    'INVERT_NEGS',
+    'MATH',
+    'NEGATE_CONDITIONALS',
+    'VOID_METHOD_CALLS',
+    'EMPTY_RETURNS',
+    'FALSE_RETURNS',
+    'NULL_RETURNS',
+    'PRIMITIVE_RETURNS',
+    'TRUE_RETURNS',
+    'REMOVE_CONDITIONALS'
+  ])
```

Comment:
Are these coming from some sort of programme wide guidelines?

Reply:
Yes Rafa